Align protocol docs with style guide (#575)
LiskHQ/lisk-docs: modules/ROOT/pages/transactions.adoc

= Transactions

> :description: The transactions section of the Lisk protocol outlines the default transactions present in the Lisk protocol.

Reviewer: ```suggestion
:description: This section describes the default transactions present in the Lisk protocol.
```

Reviewer: Two times "Lisk protocol" seems redundant and also repeating the section title seems redundant.

For example, in Lisk Mainnet, for a <<transfer,balance transfer>> transaction with a size of 133 bytes, the minimum fee is 10^-5^ LSK/byte × 133 bytes = 0.00133 LSK.
In the case of <<delegate,delegate registration>> transaction with a size of 124 bytes, the minimum fee of the transaction is 10 LSK + 10^-5^ LSK/byte × 124 bytes = 10.00124 LSK.

This required minimum fee paid by every transaction is burned, in other words, it is not assigned to any account balance.
The remaining fee on top of this is assigned to the delegate forging the block in which the transaction is included.
This implies that under normal circumstances delegates give priority to transactions with higher remaining fees.

=== `senderPublicKey`

The public key of the account issuing the transaction.
Note that this public key does not necessarily sign the transaction.
For simplicity, the account issuing the transaction is called the sender account in this document.

=== `asset`

This property contains the data specific to the type of the transaction.
For each `type` value, there exist a unique `asset` property.

> The section <<types,Transaction types>> describes the different transaction types with respect to the `asset` property of each of them.

Reviewer: I am not an expert on style (rather consult Chris), but I find it odd to reference to a section title without quotation marks. In particular, as there is suddenly upper case in the middle of the sentence.

Author: It is displayed as link, so bold and blue, imho that makes separation pretty clear.
